beacon_node/lighthouse_network/src/rpc/methods.rs

```diff
@@ -374,29 +374,21 @@ impl BlobsByRootRequest {
 
 #[derive(Debug, Clone, PartialEq)]
 pub enum RPCResponse<T: EthSpec> {
-    /// A HELLO message.
+
     Status(StatusMessage),
 
-    /// A response to a get BLOCKS_BY_RANGE request. A None response signifies the end of the
-    /// batch.
     BlocksByRange(Arc<SignedBeaconBlock<T>>),
 
-    /// A response to a get BLOCKS_BY_ROOT request.
     BlocksByRoot(Arc<SignedBeaconBlock<T>>),
 
-    /// A response to a get BLOBS_BY_RANGE request
     BlobsByRange(Arc<BlobSidecar<T>>),
 
-    /// A response to a get LIGHT_CLIENT_BOOTSTRAP request.
     LightClientBootstrap(Arc<LightClientBootstrap<T>>),
 
-    /// A response to a get BLOBS_BY_ROOT request.
     BlobsByRoot(Arc<BlobSidecar<T>>),
 
```

Comment:
You can remove the whitespace too. Also `ResponseTermination` variant comments can be dropped too